.github/workflows/github-actions.yml

```diff
@@ -61,3 +61,22 @@ jobs:
     - run: npm run electron:windows
     - run: npm run lint 
     - run: npm test -- "--karma-config=./tests/karma.ci.conf.js"
+
+  code-coverage:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps: 
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: "0"
```

Comment:
I don't think it's necessary to include a `fetch-depth of 0` to include all branches and tags :x 
I would say instead we remove this! 

Tested out your workflow changes in my own fork and it seems to work! You can have a look [here](https://github.com/seanlowjk/CATcher/actions/runs/537335757) if needed!